src/api/constants/index.ts

```diff
@@ -143,6 +143,24 @@ export const ACCOUNT = {
   MAX_AUTH_RETRIES_TIMEFRAME: DATE_24_HOURS_IN_THE_PAST(),
 };
 
+export const FINANCIAL_DETAILS = {
+  ENCRYPTION: {
+    KEY_LENGTH: 128,
+    ENCODING: 'hex' as BufferEncoding,
+    STRING_ENCODING: 'base64' as BufferEncoding,
+    ALGORITHM: 'sha512' as Algorithm,
+    ENCRYPTION_METHOD: 'aes-256-cbc',
+    OUTPUT_ENCODING: 'utf-8' as BufferEncoding,
```

Comment:
Wondering if these could be simpler, reads a little funny eg

```js
ENCRYPTION.ENCODING
ENCRYPTION.STRING_ENCODING
ENCRYPTION.ENCRYPTION_METHOD
ENCRYPTION.OUTPUT_ENCODING
```

If they're only used for specific things can we wrap them in an object? e.g

```js
ENCRYPTION.KEY.METHOD
ENCRYPTION.IV.XYZ
```